aws/endpoints/v2/ednpoints.go

```diff
@@ -0,0 +1,177 @@
+package endpoints
```

Comment:
We can actually replace the existing endpoints implementation since all API clients will be regenerated with this pattern. No need to preserve the old behavior.

Reply:
Also this file name has minor typo in it.